src/services/videoconference/logic/index.js

```diff
@@ -0,0 +1,102 @@
+const rp = require('request-promise-native');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['rp' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4819756)